backends/gpu/mlir_tests/rocm/blas.mlir

```diff
@@ -0,0 +1,193 @@
+// Copyright 2020 The TensorFlow Runtime Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// RUN: bef_executor_lite %s.bef | FileCheck %s
+
+// CHECK-LABEL: --- Running 'blas_axpy'
+func @blas_axpy() {
```

Comment:
I don't see what's changed from the cuda/blas.mlir file. I assume it's just replacing the CUDA constants with ROCm constants?